toonz/sources/toonz/iocommand.cpp

```diff
@@ -3001,6 +3002,82 @@ class OverwritePaletteCommandHandler final : public MenuItemHandler {
   }
 } overwritePaletteCommandHandler;
 
+//=============================================================================
+// Save Default palette
+//-----------------------------------------------------------------------------
+class SaveAsDefaultPaletteCommandHandler final : public MenuItemHandler {
+public:
+  SaveAsDefaultPaletteCommandHandler()
+      : MenuItemHandler(MI_SaveAsDefaultPalette) {}
+
+  void execute() override {
+    TPalette *palette = TApp::instance()->getCurrentPalette()->getPalette();
+    if (!palette) {
+      DVGui::warning("No current palette");
+      return;
+    }
+
+    int levelType = UNKNOWN_XSHLEVEL;
+    QString levelTypeStr, displayStr;
+
+    if (palette->isDefaultPalette()) {
+      levelType = palette->getDefaultPaletteType();
+    } else {
+      TXshLevel *level = TApp::instance()->getCurrentLevel()->getLevel();
+      if (!level) {
+        DVGui::warning("No current level.");
+        return;
+      }
+      TXshSimpleLevel *sl = level->getSimpleLevel();
+      if (!sl) {
+        DVGui::warning("Current level is not a simple level.");
```

Comment:
Might be more clear as "Current level is not a drawing level"?

Reply:
Will fix